keyboards/southpaw75/readme.md

```diff
@@ -0,0 +1,17 @@
+Southpaw75
+===
+
+![Southpaw75](https://imgur.com/a/dzjpDv9)
+
+60% with southpaw numpad.
+
+* Keyboard Maintainer: [kelvinhall05](https://github.com/kelvinhall05/)
+* Hardware Supported: Southpaw75 Pro Micro PCB
+* Hardware Availability: (https://github.com/kelvinhall05)
```

Comment:
```suggestion
* Hardware Availability: https://github.com/kelvinhall05
```

Ideally this should link to a public GitHub repo, or a product page where one can buy the PCB/keyboard.

Reply:
Done, changed that. Repo is currently private so it 404s but will go public after my initial group buy.